### ISSUE TITLE
Calendar: 400 bad request when using DateTime as parameter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-api-java-client#611]
```
Version of google-api-java-client (e.g. 1.5.0-beta)?
1.11.0-beta

Java environment (e.g. Java 6, Android 2.3, App Engine)?
Android 4.x

Describe the problem.
I previously used 1.9.0-beta and the corresponding generated calendar library. 
After updating I noticed that the setUpdatedMin, setTimeMin and setTimeMax 
methods from Calendar.Events.List now require DateTime objects instead of 
Strings. After changing my code I get 400 bad request errors.

Example code
DateTime lastSync = new DateTime(0);
service.events().list(calendarId).setUpdatedMin(lastSync).execute();

Stacktrace/Response
09-23 18:25:45.632: WARN/System.err(1467): 
com.google.api.client.googleapis.json.GoogleJsonResponseException: 400 Bad 
Request
09-23 18:25:45.632: WARN/System.err(1467): {
09-23 18:25:45.632: WARN/System.err(1467): "code": 400,
09-23 18:25:45.632: WARN/System.err(1467): "errors": [
09-23 18:25:45.642: WARN/System.err(1467): {
09-23 18:25:45.672: WARN/System.err(1467): "domain": "global",
09-23 18:25:45.713: WARN/System.err(1467): "message": "Bad Request",
09-23 18:25:45.713: WARN/System.err(1467): "reason": "badRequest"
09-23 18:25:45.713: WARN/System.err(1467): }
09-23 18:25:45.713: WARN/System.err(1467): ],
09-23 18:25:45.722: WARN/System.err(1467): "message": "Bad Request"
09-23 18:25:45.722: WARN/System.err(1467): }

```

Original issue reported on code.google.com by `gabriel....@gmail.com` on 23 Sep 2012 at 4:36

**Comment by GoogleCodeExporter** [comment]
```
Some more detail. It works when using the following code, so you should at 
least update the documentation for the methods that use DateTime.

Date date = new Date();
date.setTime(lastSyncDate);
DateTime lastSync = new DateTime(date, TimeZone.getDefault());
service.events().list(calendarId).setUpdatedMin(lastSync).execute();
```

Original comment by `gabriel....@gmail.com` on 27 Sep 2012 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
It seems that the important thing here is to set the TimeZone when creating the 
DateTime object, or you will get the "Bad Request" error. (i'm using verison 
1.12.0-beta)
```

Original comment by `mmmi...@gmail.com` on 5 Dec 2012 at 8:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `rmis...@google.com` on 14 Jan 2013 at 3:31

**Comment by GoogleCodeExporter** [comment]
```
I think Nick fixed this for the version 1.13 release so the time zone is always 
set.  Thanks for reporting the problem.
```

Original comment by `yan...@google.com` on 14 Jan 2013 at 5:34
- Changed state: **Fixed**
- Added labels: **Milestone-Version1.13.0**
